src/components/NoData/NoData.css.ts

```diff
@@ -0,0 +1,23 @@
+import { style } from '@vanilla-extract/css';
+import { vars } from '@/styles/theme.css';
+/**
+ * @todo 공용 폰트 스타일로 지정
+ */
```

Comment:
no data 페이지는 현재 올라온 공용 폰트로 거의 그대로 갈 것 같아서, 바로 이용해도 될 것 같아요~!!

```suggestion
import * as fonts from '@/styles/font.css';

export const message = style([fonts.titleRegular]);
```

Reply:
오오 좋아요!! 바꿔두도록 하겠습니다..!!